File: src/test/resources/input/basic/akka/persistence/testkit/EventStorage.scala
package akka.persistence.testkit

import akka.persistence.testkit.ProcessingPolicy.DefaultPolicies

private[testkit] trait EventStorage {
    import EventStorage._

    val DefaultPolicy = JournalPolicies.PassAll
}

object EventStorage {
    object JournalPolicies extends DefaultPolicies[JournalOperation]
}

sealed trait JournalOperation

File: src/test/resources/expected_output/basic/akka/persistence/testkit/EventStorage.java
package akka.persistence.testkit;
public  interface EventStorage {
  static public  class JournalPolicies$ implements akka.persistence.testkit.ProcessingPolicy.DefaultPolicies<akka.persistence.testkit.JournalOperation> {
    /**
      * Static reference to the singleton instance of this Scala object.
      */
    public static final JournalPolicies$ MODULE$ = null;
    public   JournalPolicies$ ()  { throw new RuntimeException(); }
  }
  public  akka.persistence.testkit.ProcessingPolicy.DefaultPolicies<akka.persistence.testkit.JournalOperation>.PassAll$ DefaultPolicy ()  ;
}
